### PR TITLE
Bump Stackdriver gem version in fluentd-gcp image

### PR DESCRIPTION
**Pull request by crassirostris**
To support monitoring, introduced in https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/135

### CODE REVIEW

**Comment by piosz** [comment]
cc @igorpeshansky

File: fluentd-gcp-image/Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 gem 'fluentd', '~>0.12.32'
 gem 'fluent-plugin-record-reformer', '~>0.8.3'
 gem 'fluent-plugin-systemd', '~>0.0.8'
-gem 'fluent-plugin-google-cloud', '~>0.6.3'
+gem 'fluent-plugin-google-cloud', '~>0.6.4'
```
Comment (igorpeshansky) [inline fluentd-gcp-image/Gemfile]: Don't you also need to add the Prometheus plugin?

Comment (crassirostris) [inline fluentd-gcp-image/Gemfile]: It's already there, 8th line

Comment (igorpeshansky) [inline fluentd-gcp-image/Gemfile]: Sorry, never mind.

**Pull request merged**